plugins/modules/win_domain_user.py

```diff
@@ -136,6 +136,10 @@
       - Configures the user's last name (surname).
     type: str
     aliases: [ lastname ]
+  display_name:
+    description:
+      - Configures the user's display name.
+    type: str
```

Comment:
This needs a `version_added: 1.12.0` line to denote when the option was added.

Reply:
✅ Added
Thanks!